godot/src/ui/dialogs/nft_dialog.tscn

```diff
@@ -0,0 +1,186 @@
+custom_minimum_size = Vector2(0, 340)
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="Background" type="ColorRect" parent="MainVContainer/BodyHContainer/Image"]
+unique_name_in_owner = true
+layout_mode = 1
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+color = Color(1, 1, 1, 0)
+
+[node name="NFTImage" type="TextureRect" parent="MainVContainer/BodyHContainer/Image"]
+unique_name_in_owner = true
+layout_mode = 1
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+size_flags_horizontal = 3
+expand_mode = 1
+stretch_mode = 5
+
+[node name="Separator" type="Control" parent="MainVContainer/BodyHContainer"]
+custom_minimum_size = Vector2(10, 0)
+layout_mode = 2
+
+[node name="InformationPanel" type="VBoxContainer" parent="MainVContainer/BodyHContainer"]
+unique_name_in_owner = true
+custom_minimum_size = Vector2(400, 0)
+layout_mode = 2
+
+[node name="LabelOwner" type="Label" parent="MainVContainer/BodyHContainer/InformationPanel"]
+layout_mode = 2
+theme_override_colors/font_color = Color(0.388235, 0.388235, 0.478431, 1)
+theme_override_font_sizes/font_size = 14
+text = "OWNER"
+
+[node name="Owner" type="RichTextLabel" parent="MainVContainer/BodyHContainer/InformationPanel"]
+unique_name_in_owner = true
+custom_minimum_size = Vector2(0, 20)
+layout_mode = 2
+theme_override_colors/default_color = Color(0, 0, 0, 1)
+theme_override_font_sizes/normal_font_size = 14
+bbcode_enabled = true
+fit_content = true
+
+[node name="Separator2" type="Control" parent="MainVContainer/BodyHContainer/InformationPanel"]
+custom_minimum_size = Vector2(0, 5)
+layout_mode = 2
+
+[node name="HBoxContainer" type="HBoxContainer" parent="MainVContainer/BodyHContainer/InformationPanel"]
+layout_mode = 2
+
+[node name="VBoxContainer" type="VBoxContainer" parent="MainVContainer/BodyHContainer/InformationPanel/HBoxContainer"]
+layout_mode = 2
+
+[node name="LabelLastSoldFor" type="Label" parent="MainVContainer/BodyHContainer/InformationPanel/HBoxContainer/VBoxContainer"]
+layout_mode = 2
+theme_override_colors/font_color = Color(0.388235, 0.388235, 0.478431, 1)
+theme_override_font_sizes/font_size = 14
+text = "LAST SOLD FOR"
+
+[node name="LastSoldFor" type="Label" parent="MainVContainer/BodyHContainer/InformationPanel/HBoxContainer/VBoxContainer"]
+unique_name_in_owner = true
+custom_minimum_size = Vector2(0, 20)
+layout_mode = 2
+theme_override_colors/font_color = Color(0, 0, 0, 1)
+theme_override_font_sizes/font_size = 14
+
+[node name="Separator" type="Control" parent="MainVContainer/BodyHContainer/InformationPanel/HBoxContainer"]
+layout_mode = 2
+size_flags_horizontal = 3
+
+[node name="VBoxContainer2" type="VBoxContainer" parent="MainVContainer/BodyHContainer/InformationPanel/HBoxContainer"]
+layout_mode = 2
+
+[node name="LabelAvgPrice" type="Label" parent="MainVContainer/BodyHContainer/InformationPanel/HBoxContainer/VBoxContainer2"]
```

Comment:
```suggestion
[node name="LabelAvgPrice" type="Label" parent="VBoxContainer_Main/HBoxContainer_Body/Panel_Information/HBoxContainer/VBoxContainer_Some?"]
```
don't do it if you don't want now, but the convention I was following is `NodeType_Name` => `Button_Exit`, `Label_Info`, `Panel_TestWindow`, etc